sdcm/tester.py

```diff
@@ -35,7 +35,7 @@
 
 from sdcm import nemesis, cluster_docker, cluster_baremetal, cluster_k8s, db_stats, wait
 from sdcm.cluster import NoMonitorSet, SCYLLA_DIR, Setup, UserRemoteCredentials, set_duration as set_cluster_duration, \
-    set_ip_ssh_connections as set_cluster_ip_ssh_connections, BaseLoaderSet, BaseMonitorSet, BaseScyllaCluster
+    set_ip_ssh_connections as set_cluster_ip_ssh_connections, BaseLoaderSet, BaseMonitorSet, BaseScyllaCluster  # pylint: disable=no-name-in-module
```

Comment:
why did you add this? I don't see any issue here in `master`

Reply:
the pre-commit fails for this in what looks like a python 3.8 issue:
```
pylint...................................................................Failed
- hook id: pylint
- exit code: 2

************* Module sdcm.tester
sdcm/tester.py:37:0: E0611: No name 'cluster' in module 'sdcm' (no-name-in-module)
sdcm/tester.py:37:0: E0001: Cannot import 'sdcm.cluster' due to syntax error 'invalid syntax (<unknown>, line 1514)' (syntax-error)

```

Reply:
@yarongilor  is that error from a local run of your, or from Jenkins ?